consensus/polybft/polybft_config.go

```diff
@@ -247,11 +248,25 @@ func ParseRawTokenConfig(rawConfig string) (*TokenConfig, error) {
 		return nil, errInvalidTokenParams
 	}
 
+	var chainID uint64
+
+	if !isMintable {
+		if len(params) == minNativeTokenParamsNumber+1 {
+			chainID, err = strconv.ParseUint(strings.TrimSpace(params[4]), 10, 16)
```

Comment:
I think we have a function in common package, something like ParseUint64 or something like that.